### PR TITLE
BrenKt-SK903-WEB-Domaine-Zonemaster_userguide

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web/domains/dns_zonemaster/guide.fr-fr.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: 'Utilisation de Zonemaster'
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.fr-fr.md]:
```suggestion
title: 'Tutoriel - Utilisation de Zonemaster'
```

File: pages/web/domains/dns_zonemaster/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+slug: zonecheck-de-votre-domaine
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.fr-fr.md]:
```suggestion
slug: ovhcloud-domain-zonemaster-tutorial

File: pages/web/domains/dns_zonemaster/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+order: 08
+---
+
+**Dernière mise à jour le 12/09/2022**
+
+> [!warning]
+>
+> OVHcloud met à votre disposition des services dont la configuration, la gestion et la responsabilité vous incombent. Il vous revient de ce fait d'en assurer le bon fonctionnement.
+> 
+> Nous mettons à votre disposition ce guide afin de vous accompagner au mieux sur des tâches courantes. Néanmoins, nous vous recommandons de faire appel à un [prestataire spécialisé](https://partner.ovhcloud.com/fr/) si vous éprouvez des difficultés. En effet, nous ne serons pas en mesure de vous fournir une assistance. Plus d'informations dans la section [« Aller plus loin »](#go-further) de ce guide.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.fr-fr.md]:
```suggestion
> Nous vous recommandons de faire appel à un [prestataire spécialisé](https://partner.ovhcloud.com/fr/) si vous éprouvez des difficultés. En effet, nous ne serons pas en mesure de vous fournir une assistance. Plus d'informations dans la section [« Aller plus loin »](#go-further) de ce tutoriel.
```

File: pages/web/domains/dns_zonemaster/guide.fr-fr.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+- Disposer d'un [nom de domaine](https://www.ovhcloud.com/fr/domain/)
+
+## En pratique
+
+### Champ de saisie
+
+L'outil Zonemaster permet de vérifier une configuration DNS en place sur un nom de domaine ou de tester une zone DNS préconfigurée sur des futurs serveurs DNS.
+
+Pour vérifier la configuration actuelle d'un nom de domaine, saisissez votre nom de domaine, puis cliquez sur `check`{.action}
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.fr-fr.md]:
```suggestion
Pour vérifier la configuration actuelle d'un nom de domaine, saisissez votre nom de domaine puis cliquez sur `Check`{.action}
```

File: pages/web/domains/dns_zonemaster/guide.fr-fr.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+![domains](images/zonemaster01.png){.thumbnail}
+
+Pour vérifier une configuration DNS qui a été préparée, mais pas encore appliquée au nom de domaine concerné, cochez la case `Options`{.action}, puis saisissez les informations suivantes :
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.fr-fr.md]:
```suggestion
Pour vérifier une configuration DNS que vous avez préparée mais pas encore appliquée au nom de domaine concerné, cochez la case `Options`{.action}, puis saisissez les informations suivantes :
```

File: pages/web/domains/dns_zonemaster/guide.fr-fr.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+- **Délégation du Signataire (enregistrement DS)**: Dans le cadre d'une protection DNSSEC, saisissez les éléments de l'enregistrement DS, puis cliquez sur `+`{.action} pour ajouter la valeur. Si les serveurs DNS n'utilisent pas le protocole DNSSEC, vous pouvez laisser ces champs libres.
+
+Vous pouvez également forcer les vérifications sur un protocole IP spécifique, via les cases `Désactiver IPv6` et `Désactiver IPv4`
+
+> **Exemple**:<br><br> Vous possédez le nom de domaine « mydomain.ovh » qui utilise actuellement les serveurs DNS « dns19.ovh.net » et  « ns19.ovh.net ». Vous avez configuré une zone DNS pour ce nom domaine sur les serveurs DNS « mydns.test.ovh » et « mydns2.test.ovh ». Avant de changer les serveurs DNS, vous pouvez effectuez une recherche avancée à la l'aide de la case `Options`{.action} en saisissant « mydns.test.ovh » et « mydns2.test.ovh » dans les cases `Serveurs DNS`. Zonemaster réalisera un test comme si vous utilisiez les serveurs « mydns.test.ovh » et « mydns2.test.ovh » sur « mydomain.ovh ».<br>
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.fr-fr.md]:
```suggestion
> **Exemple**:<br><br> Vous possédez le nom de domaine « mydomain.ovh » qui utilise actuellement les serveurs DNS « dns19.ovh.net » et  « ns19.ovh.net ».
>
>Vous avez configuré une zone DNS pour ce nom domaine sur les serveurs DNS « mydns.test.ovh » et « mydns2.test.ovh ».<br>
> Avant de changer les serveurs DNS, vous pouvez effectuer une recherche avancée à la l'aide de la case `Options`{.action} en saisissant « mydns.test.ovh » et « mydns2.test.ovh » dans les cases `Serveurs DNS`.<br>
> Zonemaster réalisera un test comme si vous utilisiez les serveurs « mydns.test.ovh » et « mydns2.test.ovh » sur « mydomain.ovh ».<br>
```

File: pages/web/domains/dns_zonemaster/guide.fr-fr.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+> ![domains](images/zonemaster02.png){.thumbnail}
+
+> [!primary]
+>
+> Lorsque vous saisissez un nom de domaine et que vous cliquez sur le bouton `Obtenir les données de la zone parente`{.action}, les serveurs DNS associés au nom de domaine apparaitront ainsi que les informations de l'enregistrement DS (DNSSEC) si celui-ci a été configuré.
+> ![domains](images/zonemaster03.png){.thumbnail}
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.fr-fr.md:58]:
```suggestion
>
> ![domains](images/zonemaster03.png){.thumbnail}
```

File: pages/web/domains/dns_zonemaster/guide.fr-fr.md [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+
+
+### Résultat
+
+Une fois le formulaire validé, les résultats sont classés par code couleur :
+
+- **Vert** : Cette partie est fonctionnelle et répond aux critères standard dans sa catégorie.
+- **Orange** : Cette partie est fonctionnelle, mais mérite une attention particulière. L'outil a détecté que ce paramètre présente des caractéristiques qui ne rentrent pas au standard de sa catégorie, sans bloquer son fonctionnement.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.fr-fr.md]:
```suggestion
- **Orange** : Cette partie est fonctionnelle, mais mérite une attention particulière. L'outil a détecté qu'un paramètre ne répond pas au standard de sa catégorie, sans que cela ne bloque son fonctionnement.
```

File: pages/web/domains/dns_zonemaster/guide.fr-fr.md [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+- **Rouge** : Cette partie présente des erreurs ou des éléments manquants pouvant causer un dysfonctionnement  . 
+- **Bleu** : il s'agit d'une simple information, sans conséquence particulière sur le fonctionnement du nom de domaine.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.fr-fr.md]:
```suggestion
- **Bleu** : Il s'agit d'une simple information, sans conséquence particulière sur le fonctionnement du nom de domaine.
```

File: pages/web/domains/dns_zonemaster/guide.fr-fr.md [hunk 10]
```diff
@@ -0,0 +1,84 @@
...
+### Informations utiles
+
+Si vous avez des questions supplémentaires au sujet de Zonemaster, consultez la rubrique [FAQ](https://zonemaster.net/faq) sur <https://zonemaster.fr/>.
+
+
+## Aller plus loin
+
+[Généralités sur les serveurs DNS](../generalites-serveurs-dns/){.external}.
+
+[Modification d'une zone DNS](https://docs.ovh.com/fr/domains/editer-ma-zone-dns/){.external} DNS OVHcloud.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.fr-fr.md]:
```suggestion
[Modification d'une zone DNS OVHcloud](https://docs.ovh.com/fr/domains/editer-ma-zone-dns/){.external}.
```

File: pages/web/domains/dns_zonemaster/guide.en-gb.md [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+> We have provided you with this guide in order to help you with common tasks. Nevertheless, we recommend contacting a specialist [provider](https://partner.ovhcloud.com/en-gb/) if you encounter any difficulties. We will not be able to assist you ourselves. You can find more information in the [Go further”](#go-further) section of this guide.
+> 
+
+## Objective
+
+[Zonemaster](https://zonemaster.net/domain_check) is a tool created by the collaboration between [AFNIC](https://www.afnic.fr/en/) (French registry) and [The Swedish Internet Foundation](https://internetstiftelsen.se/en/) (Swedish registry). It allows you to analyse the Domain Name System (DNS) configuration of a domain name and identify what can be improved or corrected.
+
+> [!primary]
+>
+> To get a better understanding of DNS, please read the introduction to our guide on [configuring a DNS](https://docs.ovh.com/gb/en/domains/web_hosting_how_to_edit_my_dns_zone/) zone.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.en-gb.md]:
```suggestion
> To get a better understanding of DNS, please read the introduction to our guide on [configuring a DNS zone](https://docs.ovh.com/gb/en/domains/web_hosting_how_to_edit_my_dns_zone/).
```

File: pages/web/domains/dns_zonemaster/guide.en-gb.md [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+
+- a [domain name](https://www.ovhcloud.com/en-gb/domains/)
+
+## Instructions
+
+### Input field
+
+The Zonemaster tool allows you to check a DNS configuration in place on a domain name or to test a preconfigured DNS zone on future DNS servers.
+
+To check the current configuration of a domain name, enter your domain name, then click `check`{.action}
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.en-gb.md]:
```suggestion
To check the current configuration of a domain name, enter your domain name, then click `Check`{.action}
```

File: pages/web/domains/dns_zonemaster/guide.en-gb.md [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+
+![domains](images/zonemaster01.png){.thumbnail}
+
+To check a DNS configuration that has been prepared but not yet applied to the domain name concerned, select the `Options`{.action} box, and then enter the following information:
+
+- **DNS** servers: Enter the information for the DNS server associated with a domain name, then click the `+`{.action} button to confirm your entry. Entering an IP address is optional.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.en-gb.md]: please check options names on EN version of Zonemaster

File: pages/web/domains/dns_zonemaster/guide.en-gb.md [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+- **Delegation of the Signatory (DS record)**: For DNSSEC protection, enter the DS record items, and then click `+`{.action} to add the value. If the DNS servers do not use the DNSSEC protocol, you can leave these fields free.
+
+You can also force checks on a specific IP protocol by using the `Disable IPv6` and `Disable IPv4 checkboxes`
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.en-gb.md]:
```suggestion
You can also force checks on a specific IP protocol by using the `Disable IPv6` and `Disable IPv4` checkboxes.
```

File: pages/web/domains/dns_zonemaster/guide.en-gb.md [hunk 15]
```diff
@@ -0,0 +1,86 @@
...
+- **Green**: This part is functional and meets the standard criteria in its category.
+- **Orange**: This part is functional, but deserves special attention. The tool has detected that this parameter has characteristics that do not fit within its category, without blocking its operation.
+- **Red**: This part contains errors or missing elements that may cause a malfunction. 
+- **Blue**: this is simple information, with no particular consequences for the functioning of the domain name.
+
+![domains](images/zonemaster04.png){.thumbnail}
+
+### Useful information
+
+If you have any additional questions about Zonemaster, see [FAQ](https://zonemaster.net/faq) on <https://zonemaster.fr/>.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.en-gb.md]:
```suggestion
If you have any additional questions about Zonemaster, see the [FAQ](https://zonemaster.net/faq) on <https://zonemaster.fr/>.
```

File: pages/web/domains/dns_zonemaster/guide.es-es.md [hunk 16]
```diff
@@ -0,0 +1,85 @@
+---
+title: 'Uso de Zonemaster'
+slug: ovhcloud-domain-zonemaster-tutorial
+section: DNS servidor y zona
+order: 08
+---
+
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.es-es.md:7]: Systran warning is missing

File: pages/web/domains/dns_zonemaster/guide.es-es.md [hunk 17]
```diff
@@ -0,0 +1,85 @@
...
+- **Rojo**: Esta parte presenta errores o elementos que faltan y que pueden provocar un fallo de funcionamiento. 
+- **Azul**: se trata de una simple información, sin consecuencias particulares sobre el funcionamiento del nombre de dominio.
+
+![dominios](images/zonemaster04.png){.thumbnail}
+
+### Información útil
+
+Si tiene más preguntas sobre Zonemaster, consulte [FAQ](https://zonemaster.net/faq) en <https://zonemaster.fr/>.
+
+## Más información
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.es-es.md]: missing anchor link

File: pages/web/domains/dns_zonemaster/guide.es-es.md [hunk 18]
```diff
@@ -0,0 +1,85 @@
...
+
+- Tener un [dominio](https://www.ovhcloud.com/es-es/domains/).
+
+## Procedimiento
+
+### Campo de entrada
+
+La herramienta Zonemaster permite comprobar una configuración DNS en un dominio o probar una zona DNS preconfigurada en futuros servidores DNS.
+
+Para comprobar la configuración actual de un dominio, introduzca el dominio y haga clic en `check`{.action}.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.es-es.md]:
```suggestion
Para comprobar la configuración actual de un dominio, introduzca el dominio y haga clic en `Check`{.action}.
```

File: pages/web/domains/dns_zonemaster/guide.es-es.md [hunk 19]
```diff
@@ -0,0 +1,85 @@
...
+>
+> Al introducir un dominio y hacer clic en el botón `Obtener datos desde la zona padre`{.action}, aparecerán los servidores DNS asociados al dominio y la información del registro DS (DNSSEC) si este se ha configurado.
+> ![dominios](images/zonemaster03.png){.thumbnail}
+
+### Resultado
+
+Una vez validado el formulario, los resultados se clasifican por código de color:
+
+- **Verde**: Esta parte es funcional y cumple los criterios estándar en su categoría.
+- **Orange**: Esta parte es funcional, pero merece una atención especial. La herramienta ha detectado que este parámetro tiene características que no encajan en el estándar de su categoría, sin bloquear su funcionamiento.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.es-es.md]:
```suggestion
- **Naranja**: Esta parte es funcional, pero merece una atención especial. La herramienta ha detectado que este parámetro tiene características que no encajan en el estándar de su categoría, sin bloquear su funcionamiento.
```

File: pages/web/domains/dns_zonemaster/guide.it-it.md [hunk 20]
```diff
@@ -0,0 +1,86 @@
+---
+title: Utilizzo di Zonemaster
+slug: ovhcloud-domain-zonemaster-tutorial
+section: DNS e zona DNS
+order: 08
+---
+
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.it-it.md:7]: Systran warning is missing

File: pages/web/domains/dns_zonemaster/guide.it-it.md [hunk 21]
```diff
@@ -0,0 +1,86 @@
...
+- Disporre di un [dominio](https://www.ovhcloud.com/fr/domain/)
+
+## Procedura
+
+### Campo di inserimento
+
+Il tool Zonemaster permette di verificare una configurazione DNS installata su un dominio o di testare una zona DNS preconfigurata su server DNS futuri.
+
+Per verificare l'attuale configurazione di un dominio, inserisci il tuo dominio e clicca su `check`{.action}
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.it-it.md]:
```suggestion
Per verificare l'attuale configurazione di un dominio, inserisci il tuo dominio e clicca su `Check`{.action}
```

File: pages/web/domains/dns_zonemaster/guide.it-it.md [hunk 22]
```diff
@@ -0,0 +1,86 @@
...
+
+![domini](images/zonemaster01.png){.thumbnail}
+
+Per verificare una configurazione DNS preparata ma non ancora applicata al dominio interessato, seleziona la casella `Options`{.action} e inserisci le seguenti informazioni:
+
+- **Server DNS**: Inserisci le informazioni del server DNS associato a un dominio e clicca su `+`{.action} per confermare l'inserimento. Inserire un indirizzo IP è facoltativo.
+- **Delega del firmatario (registrazione DS)**: In caso di protezione DNSSEC, inserisci gli elementi del record DS e clicca su `+`{.action} per aggiungere il valore. Se i server DNS non utilizzano il protocollo DNSSEC, è possibile lasciare liberi questi campi.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.it-it.md:47]: Please use EN wording as no IT translation is available

File: pages/web/domains/dns_zonemaster/guide.it-it.md [hunk 23]
```diff
@@ -0,0 +1,86 @@
...
+
+![domini](images/zonemaster04.png){.thumbnail}
+
+### Informazioni utili
+
+Se hai altre domande su Zonemaster, consulta la sezione [FAQ](https://zonemaster.net/faq) su <https://zonemaster.fr/>.
+
+## Per saperne di più
+
+[Modificare i server DNS di un dominio OVH](https://docs.ovh.com/it/domains/web_hosting_gestisci_il_tuo_server_dns/){.external}
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.it-it.md:80]:
```suggestion
[Modificare i server DNS di un dominio OVHcloud](https://docs.ovh.com/it/domains/web_hosting_gestisci_il_tuo_server_dns/){.external}
```

File: pages/web/domains/dns_zonemaster/guide.pl-pl.md [hunk 24]
```diff
@@ -0,0 +1,85 @@
+---
+title: 'Korzystanie z opcji Zonemaster'
+slug: ovhcloud-domain-zonemaster-tutorial
+section: DNS i strefa DNS
+order: 08
+---
+
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.pl-pl.md:7]: Systran warning is missing

File: pages/web/domains/dns_zonemaster/guide.pl-pl.md [hunk 25]
```diff
@@ -0,0 +1,85 @@
...
+Narzędzie Zonemaster umożliwia sprawdzenie konfiguracji DNS w Twojej domenie lub przetestowanie strefy DNS na przyszłych serwerach DNS.
+
+Aby sprawdzić aktualną konfigurację domeny, wpisz nazwę domeny, następnie kliknij `Check`{.action}
+
+![domeny](images/zonemaster01.png){.thumbnail}
+
+Aby sprawdzić konfigurację DNS, która została przygotowana, ale nie została jeszcze zastosowana dla danej domeny, zaznacz kratkę `Options`{.action}, następnie wprowadź następujące informacje:
+
+- **Serwery DNS**: Wpisz informacje dotyczące serwera DNS przypisanego do domeny, następnie kliknij na `+`{.action}, aby zatwierdzić wprowadzone dane. Wpisanie adresu IP jest opcjonalne.
+- **Delegacja Podpisującego (rejestracja DS)**: W ramach ochrony DNSSEC, wprowadź elementy rekordu DS, następnie kliknij `+`{.action}, aby dodać wartość. Jeśli serwery DNS nie używają protokołu DNSSEC, możesz zostawić te pola wolne.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.pl-pl.md:47]: Please use EN wordings

File: pages/web/domains/dns_zonemaster/guide.pt-pt.md [hunk 26]
```diff
@@ -0,0 +1,86 @@
+---
+title: 'Utilização de Zonemaster'
+slug: ovhcloud-domain-zonemaster-tutorial
+section: DNS e zona DNS
+order: 08
+---
+
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.pt-pt.md:7]: Systran warning is missing

File: pages/web/domains/dns_zonemaster/guide.pt-pt.md [hunk 27]
```diff
@@ -0,0 +1,86 @@
...
+> 
+
+
+## Objetivo
+
+[Zonemaster](https://zonemaster.fr/) é uma ferramenta originada pela colaboração entre a [AFNIC](https://www.afnic.fr/) (registo francês) e a [The Swedish Internet Foundation](https://internetstiftelsen.se/en/) (registo sueco). Permite analisar a configuração DNS (Domain Name System) de um nome de domínio e identificar os elementos que podem ser melhorados ou corrigidos.
+
+> [!primary]
+>
+> Para compreender melhor a noção de DNS, consulte a introdução do nosso guia sobre a [configuração de uma zona DNS](https://docs.ovh.com/fr/domains/editer-ma-zone-dns/).
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.pt-pt.md]: there should be a PT link

File: pages/web/domains/dns_zonemaster/guide.pt-pt.md [hunk 28]
```diff
@@ -0,0 +1,86 @@
...
+
+## Requisitos
+
+- Ter um [domínio](https://www.ovhcloud.com/fr/domain/)
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.pt-pt.md]: There should be a PT link

File: pages/web/domains/dns_zonemaster/guide.pt-pt.md [hunk 29]
```diff
@@ -0,0 +1,86 @@
...
+A ferramenta Zonemaster permite verificar uma configuração DNS instalada num nome de domínio ou testar uma zona DNS pré-configurada em futuros servidores DNS.
+
+Para verificar a configuração atual de um nome de domínio, introduza o seu nome de domínio e clique em `Check`{.action}
+
+![dominios](images/zonemaster01.png){.thumbnail}
+
+Para verificar uma configuração DNS que foi preparada, mas ainda não aplicada ao domínio em causa, selecione a opção `Options`{.action} e introduza as seguintes informações:
+
+- **Servidores DNS**: Introduza as informações do servidor DNS associado a um nome de domínio e depois clique em `+`{.action} para validar a sua introdução. A introdução de um endereço IP é facultativa.
+- **Delegação do Signatário (registo DS)**: No quadro de uma proteção DNSSEC, introduza os elementos do registo DS e clique em `+`{.action} para adicionar o valor. Se os servidores DNS não utilizarem o protocolo DNSSEC, pode deixar estes campos livres.
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.pt-pt.md:47]: Please use EN wordings

File: pages/web/domains/dns_zonemaster/guide.en-gb.md (hunk 13 above)
Comment (BrenKt) [reply pages/web/domains/dns_zonemaster/guide.en-gb.md]: same as guide

File: pages/web/domains/dns_zonemaster/guide.de-de.md [hunk 30]
```diff
@@ -0,0 +1,89 @@
+---
+title: Verwendung von Zonemaster
+slug: ovhcloud-domain-zonemaster-tutorial
+order: 08
+---
```
Comment (OVH-Team-Guides) [inline pages/web/domains/dns_zonemaster/guide.de-de.md:6]: There should be a section in the frontmatter

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**